src/main.scss

```diff
@@ -1,8 +1,8 @@
-@import '~hds-design-tokens/lib/all.scss';
-@import '~leaflet/dist/leaflet';
-@import '~leaflet-draw/dist/leaflet.draw';
-@import '~leaflet-geosearch/dist/geosearch';
-@import '~leaflet-measure-path/leaflet-measure-path';
+@import '../node_modules/hds-design-tokens/lib/all.scss';
+@import '../node_modules/leaflet/dist/leaflet';
+@import '../node_modules/leaflet-draw/dist/leaflet.draw';
+@import '../node_modules/leaflet-geosearch/dist/geosearch';
+@import '../node_modules/leaflet-measure-path/leaflet-measure-path';
```

Comment:
I tested leaving the first part out altogether and it worked on my machine at least. I'm not sure if it was that way from the beginning and if the tildes were just completely redundant.

Reply:
`Invalid dependencies have been reported by plugins or loaders for this module. All reported dependencies need to be absolute paths.
Invalid dependencies may lead to broken watching and caching.
As best effort we try to convert all invalid values to absolute paths and converting globs into context dependencies, but this is deprecated behavior.
Loaders: Pass absolute paths to this.addDependency (existing files), this.addMissingDependency (not existing files), and this.addContextDependency (directories).
Plugins: Pass absolute paths to fileDependencies (existing files), missingDependencies (not existing files), and contextDependencies (directories).
Globs: They are not supported. Pass absolute path to the directory as context dependencies.
The following invalid values have been reported:
 * "node_modules/"
 * "node_modules/hds-design-tokens"
 * "node_modules/hds-design-tokens/lib"
 * and more ...`
 
 To me the eslint says that.
 
 And also webpack compiler whines about that.

Reply:
Oh, you're right, I didn't see that appear in the console log. That message gave me a hint though: changing the SASS_PATH in the .env file to `SASS_PATH=/node_modules:/src` (so initial forward slashes added) seems to fix that part as well.